src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -30,6 +30,7 @@ const notifications_util = require('../util/notifications_util');
  */
 async function validate_input_types(type, action, argv) {
     validate_type_and_action(type, action);
+    validate_no_leading_space(argv._);
```

Comment:
Why do you call the new function second and not first?

Reply:
Here we want to validate the `time` and `action` args first so that we need not to do that in our new function and we can simply check the length

cc @romayalon

Reply:
Sure (I guess you meant `type` and not `time`).
As long as the case that was discussed is covered -` _: [ 'account', 'tmp/' ]` (you can add it as a case).